src/lib/containers/widgets/facet-nav/FacetNav.tsx

```diff
@@ -200,7 +200,7 @@ const FacetNav: React.FunctionComponent<FacetNavProps> = ({
   } else {
     return (
       <>
-        <div className={`FacetNav ${showFacetVisualization ? '' : 'hidden'}`}>
+        <div className={`FacetNav ${showFacetVisualization ? '' : 'hidden'}${showMoreState === 'LESS' ? 'less' : ''}`}>
```

Comment:
I think you need a space here.  Otherwise, if showFacetVisualization=false and showMoreState='LESS' then we'll end up with the styles 'FacetNav hiddenless'

Reply:
@conundrumgirl  explained it to me, I was confused by the showMoreState.  showMoreState is equal to 'LESS' when all the graphs are showing (indicating that's what the button would indicate).
The bug that I point out above would be a tricky state to get into, but it would happen.  User needs to click show more (show all graphs), and then hide the visualization.